quantum/quantum_random.py

```diff
@@ -0,0 +1,28 @@
+from projectq import MainEngine
+from projectq.ops import H, Measure
+
+
+def get_random_number(quantum_engine: projectq.cengines._main.MainEngine) -> int:
+    qubit = quantum_engine.allocate_qubit()
+    H | qubit
+    Measure | qubit
+    random_number = int(qubit)
+    return random_number
```

Comment:
The function signature already tells the caller that the return value is a random number.
```suggestion
    return int(qubit)
```